docs/berkeley-upgrade/migrating-archive-database-to-berkeley.mdx

```diff
@@ -60,6 +57,7 @@ mina-berkeley-migration-script  \
    --blocks-bucket mina_network_block_data \
    --blocks-batch-size 50 \
    --checkpoint-output-path .\
+   --checkpoint-interval 10000 \
```

Comment:
```suggestion
   --checkpoint-interval 100 \
```

didn't we observe issues with large interval?

Reply:
The issue is with batch-size of `berkeley_migration` app, this is the interval for `replayer`. It's different. I would advice to have the interval to be 10000

Reply:
We have tried multiple migrations. We noticed that `10000` speeds up the process considerably without impacting the performance.